docs/interact/programmability/lua-api.md

```diff
@@ -306,16 +296,15 @@ Please refer to the [Data type conversion](#data-type-conversion) for more infor
 
 ### <a name="redis.set_repl"></a> `redis.set_repl(x)`
 
-* Since version: 3.2.0
 * Available in scripts: yes
 * Available in functions: no
 
 **Note:**
 this feature is only available when script effects replication is employed.
 Calling it when using verbatim script replication will result in an error.
-As of Redis version 2.6.0, scripts were replicated verbatim, meaning that the scripts' source code was sent for execution by replicas and stored in the AOF.
-An alternative replication mode added in version 3.2.0 allows replicating only the scripts' effects.
-As of Redis version 7.0, script replication is no longer supported, and the only replication mode available is script effects replication.
+Verbatim script replication, meaning that the scripts' source code was sent for execution by replicas and stored in the AOF.
```

Comment:
I think we should rephrase this part statement, Since Redis 7.0, verbatim replication is no longer supported, I think we do not need to mention verbatim replication anymore